src/kernel/arch/aarch64/mmio.zig

```diff
@@ -1,31 +1,87 @@
-const GPIO_OFFSET: usize = 0x200000;
-const UART_OFFSET: usize = GPIO_OFFSET + 0x1000;
+const log = @import("../../log.zig");
+const std = @import("std");
+
+const MMIO_OFFSET: usize = 0x200000;
+const GPIO_OFFSET: usize = MMIO_OFFSET + 0x0000;
+const UART_OFFSET: usize = MMIO_OFFSET + 0x1000;
+const AUX_OFFSET: usize = MMIO_OFFSET + 0x15000;
```

Comment:
Nice. Making them relative to mmio_offset is more correct.

Reply:
Thanks.